translations/translations.js

```diff
@@ -34,7 +34,8 @@ let translations = {
 	hu: "translations/hu.json", // Hungarian
 	is: "translations/is.json", // Icelandic
 	et: "translations/et.json", // Estonian
-	kr: "translations/kr.json", // Korean
+	ko: "translations/ko.json", // Korean
+	kr: "translations/kr.json", // Wrong File for Korean, This code is reserved for Kanurian, but for backward compatibility; it needs keeping a while; this would be eliminated or replaced with Kanuri in future.
```

Comment:
`kr` could just point to `translations/ko.json` as well. This way we don't need to keep the two files.

And to be honest. You can just remove the 'ko' reference as well if we make note of it in the changelog. It's an easy fix for users who use the wrong language code.